### DISCUSSION TITLE
Issue With Sphinx Design Tabs (Starting in 2022.06.04)

### DISCUSSION THREAD

**Discussion by rmorshea** [pradyunsg/furo#607] [Potential Issue] [answered]
Starting with Furo version 2022.06.04 I noticed a problem with [Sphinx Design](https://sphinx-design.readthedocs.io/en/latest/)'s `tab-set` directive. Here's a side-by-side comparison with the prior version (2022.04.07):

<table>
  <tr>
    <th>2022.04.07</th>
    <th>2022.06.04</th>
  </tr>
  <tr>
    <td><img src="https://user-images.githubusercontent.com/4041990/210011875-fdc94252-2619-469d-8b32-ad80122f7b42.png" /></td>
    <td><img src="https://user-images.githubusercontent.com/4041990/210011796-38f027f7-9a0c-4f1e-930e-d203ac5bc8d5.png" /></td>
  </tr>
</table>

Somehow, the following paragraphs in the section are getting swallowed up by the `<div class="sd-tab-set docutils">` element.

<table>
  <tr>
    <th>2022.04.07</th>
    <th>2022.06.04</th>
  </tr>
  <tr>
    <td><img src="https://user-images.githubusercontent.com/4041990/210013244-49ce54a1-39d8-4c80-b48f-78dcf874bed0.png" /></td>
    <td><img src="https://user-images.githubusercontent.com/4041990/210013330-3d7a29a2-25f4-432b-aafa-40a3ca4cef05.png" /></td>
  </tr>
</table>

I glanced through the [diff](https://github.com/pradyunsg/furo/compare/2022.04.07...2022.06.04) between the two versions and wasn't able to quickly identify anything that might be causing this. The fact that the HTML itself is different suggests there's some transformation that's improperly modifying the tabs.

**Comment by pradyunsg** [comment, marked as answer]
If the HTML is different, that's likely unrelated to Furo.

**Reply by rmorshea** [reply]
Apologies, I just narrowed it down to one of my own extensions.
